src/recommendations/setting-cookie.md

```diff
@@ -0,0 +1,25 @@
+---
+group: product-recommendations
+title: Handle Cookie Restrictions
+ee_only: True
+---
+
+When you deploy the `magento/product-recommendations` module to production, the module tracks user events. These events enable behavioral data collection as long as the [cookie restriction mode](https://docs.magento.com/user-guide/stores/compliance-cookie-restriction-mode.html) is disabled. If you have custom code to handle the cookie restriction mode on your site, make sure you update your code to look for and handle the `mg_dnt` cookie.
```

Comment:
```suggestion
Magento has a native feature that allows the site to ask for cookie consent from its shoppers before storing any data in browser cookies. This feature is called [cookie restriction mode](https://docs.magento.com/user-guide/stores/compliance-cookie-restriction-mode.html). 

When you deploy the `magento/product-recommendations` module to production, it starts collecting shopper interaction events on your storefront. Because data for these events can be stored in browser cookies or localstorage, the feature supports [cookie restriction mode](https://docs.magento.com/user-guide/stores/compliance-cookie-restriction-mode.html) by not collecting events until the shopper has given cookie consent. 

This may not work with 3rd party cookie consent solutions. It is the responsibility of each merchant to ensure that data collection does not occur before cookie consent has been given, as often required by law.  If you have custom code to manage cookie consent on your site, you can use a do-not-track cookie called `mg_dnt` to restrict Magento data collection appropriately. 
```

Reply:
~~localstorage~~ > on local storage

You don't need to repeat the link.

~~3rd party~~ > third-party